CHANGELOG.md

```diff
@@ -19,8 +19,8 @@ All notable changes to this project will be documented in this file. The format
   - DMS v14 mistakenly relocated the _getmail state directory_ to the _DMS Config Volume_ as a `getmail/` subdirectory.
     - This has been corrected to `/var/lib/getmail` (_if you have mounted a DMS State Volume to `/var/mail-state`, `/var/lib/getmail` will be symlinked to `/var/mail-state/lib-getmail`_).
     - To preserve this state when upgrading to DMS v15, **you must manually migrate `getmail/` from the _DMS Config Volume_ to `lib-getmail/` in the _DMS State Volume_.**
-  - Added missing `debug getmail` subcommand to `setup` ([#4346](https://github.com/docker-mailserver/docker-mailserver/pull/4346))
-- **removed `VERSION`** file that was used for checking version updates ([#3677](https://github.com/docker-mailserver/docker-mailserver/issues/3677),[#4321](https://github.com/docker-mailserver/docker-mailserver/pull/4321))
+- **removed `VERSION` file** that was used for checking version updates ([#3677](https://github.com/docker-mailserver/docker-mailserver/issues/3677),[#4321](https://github.com/docker-mailserver/docker-mailserver/pull/4321))
+- **updated container restart behavior** to include checks and more permission adjustments ([#4323](https://github.com/docker-mailserver/docker-mailserver/pull/4323))
```

Comment:
```suggestion
- **Removed `VERSION` file** that was used for checking version updates ([#3677](https://github.com/docker-mailserver/docker-mailserver/issues/3677),[#4321](https://github.com/docker-mailserver/docker-mailserver/pull/4321))
- **Updated container restart behavior** to include checks and more permission adjustments ([#4323](https://github.com/docker-mailserver/docker-mailserver/pull/4323))
```

I'm not sure that either of these are considered breaking?

- `VERSION` removal is breaking in the sense of it affects all releases [prior to v13](https://github.com/docker-mailserver/docker-mailserver/pull/3666).
- The updated restart behaviour is more of a regression fix from v14 changes.